### PR TITLE
Kwargs change (Version 1.0.0 release)

### PR DESCRIPTION
**Pull request by ChrisKnott**
This is a breaking change to the API for start().

At the moment there is a horrible mixture of some options being specified in `options` dictionary, and some options being specified as keyword arguments.

There is no logic between what is a kwarg and what is in `options` dict.

See as in Python the kwargs are essentially just a dict anyway, this change moves every option to be a kwarg and loses `options`. I think it also makes the code nicer and better suited for future additions.

I also changed `chromeFlags` to `cmdline_args` as nothing else uses FirstCaps, and also other modes can use this option, not just Chrome.

Bumped version to 1.0.0 as I hope this is last breaking change to the interface.

### CODE REVIEW

File: eel/__init__.py [hunk 1]
```diff
@@ -85,47 +91,31 @@ def init(path, allowed_extensions=['.js', '.html', '.txt', '.htm', '.xhtml']):
...
-    _start_geometry['default'] = {'size': size, 'position': position}
-    _start_geometry['pages'] = geometry
-
-    if options['port'] == 0:
+    _start_args.update(kwargs)
+
+    if _start_args['port'] == 0:
         sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         sock.bind(('localhost', 0))
         options['port'] = sock.getsockname()[1]
```
Comment (samuelhwilliams) [inline eel/__init__.py]: Need to fix this up to `_start_args`

File: eel/browsers.py [hunk 2]
```diff
@@ -1,7 +1,16 @@
+import subprocess as sps
 import webbrowser as wbr
+
 import eel.chrome as chm
 import eel.electron as ele
-import subprocess as sps
+#import eel.edge as edg         TODO (ready to merge)
```
Comment (samuelhwilliams) [inline eel/browsers.py]: What are these about?

File: eel/electron.py [hunk 3]
```diff
@@ -2,10 +2,33 @@
 import sys
 
 def run(options, start_urls):
-	# TODO: Find Eletron properly, e.g. on Windows
-	# TODO: check for package.json and print error
+    # TODO: this properly
+    cmd = ['C:\\Users\\Chris\\Coding\\Algojammer\\node_modules\\electron\\dist\\electron.exe']
```
Comment (samuelhwilliams) [inline eel/electron.py]: Pretty sure we can't merge it like this :)

File: eel/browsers.py (hunk 2 above)
Comment (ChrisKnott) [reply eel/browsers.py]: Just as a sort of guideline for how further browsers should be supported. You can bin it if you want

File: eel/electron.py (hunk 3 above)
Comment (samuelhwilliams) [reply eel/electron.py]: Nvm, superseded by a later commit.

**Pull request merged**